avm/res/compute/virtual-machine/modules/nic-configuration.bicep

```diff
@@ -27,51 +27,40 @@ param diagnosticSettings diagnosticSettingType
 @description('Optional. Array of role assignments to create.')
 param roleAssignments roleAssignmentType
 
-module networkInterface_publicIPAddresses 'br/public:avm/res/network/public-ip-address:0.4.1' = [
-  for (ipConfiguration, index) in ipConfigurations: if (contains(ipConfiguration, 'pipConfiguration')) {
+var publicIpNameSuffix = '-pip-01'
```

Comment:
Not 100% sure about this hardcoded value here. Shouldn't this come as an input parameter? At least we should document somewhere that if neither name nor suffix is provided this default value will be used. Thoughts?

Reply:
removed